tests/behaviour/steps/params.rs

```diff
@@ -264,7 +264,7 @@ impl FromStr for IsEmptyOrNot {
 }
 
 #[derive(Debug, Parameter)]
-#[param(name = "contains_or_doesnt", regex = "(contain|do not contain)")]
+#[param(name = "contains_or_doesnt", regex = "(contain|contains|do not contain|does not contain)")]
```

Comment:
I got tired of `*thing* contain` and `*thing* do not`......... Plus, I've written drivers documents steps with `contains` and `does not contain`, so I decided to reuse the same steps for core.